python/cugraph/cugraph/tests/structure/test_graph_mg.py

```diff
@@ -99,13 +99,13 @@ def test_nodes_functionality(dask_client, input_combo):
     expected_nodes = (
         dask_cudf.concat([ddf["src"], ddf["dst"]])
         .drop_duplicates()
-        .to_frame()
-        .sort_values(0)
+        .to_frame(name="0")
+        .sort_values("0")
```

Comment:
Another "precaution" (using numerical column names still seems "fragile" in dask)